src/app/footer/footer.component.html

```diff
@@ -1,19 +1,26 @@
-<footer class="bg-slate-700 text-white">
-    <p>footer works!</p>
-    <div class="mx-auto max-w-7xl overflow-hidden px-6 py-20 sm:py-24 lg:px-8">
+<footer class="bg-blue-900 text-white">
+    <div class="mx-auto max-w-7xl overflow-hidden py-6 lg:px-8">
 
-        <nav class="-mb-6 columns-1 text-center sm:flex sm:justify-center sm:space-x-12" aria-label="Footer">
+        <nav class="-mb-6 columns-1 text-center grid grid-cols-4" aria-label="Footer">
           <div class="pb-6">
-            <a href="#" class="text-sm leading-6 text-gray-600 hover:text-gray-900">Home</a>
+            <a href="#" class="">
+              <img class="w-8 h-auto mx-auto" src="/assets/images/onboarding-images/home.png" />
+            </a>
           </div>
           <div class="pb-6">
-            <a href="#" class="text-sm leading-6 text-gray-600 hover:text-gray-900">Map</a>
+            <a href="#" class="">
+              <img class="w-10 h-auto mx-auto" src="/assets/images/onboarding-images/quest.png" />
+            </a>
           </div>
           <div class="pb-6">
-            <a href="#" class="text-sm leading-6 text-gray-600 hover:text-gray-900">Profile</a>
+            <a href="#" class="">
+              <img class="w-8 h-auto mx-auto" src="/assets/images/onboarding-images/biotool.png" />
```

Comment:
@theJayBecker <img src="smiley.gif" **_alt="_**HTML tutorial" 
add alt texts, in future samples,